bin/i18n/sync-up.rb

```diff
@@ -16,13 +16,19 @@
 # can be specified from the environment variables. We set I18N_SOURCE_DIR as an environment variable so it can be
 # globally changed for all Crowdin projects.
 ENV['I18N_SOURCE_DIR'] = CDO.dir(I18N_SOURCE_DIR)
+
+EXCLUDED_PROJECTS = %i[codeorg-restricted codeorg-restricted-test].freeze
+
```

Comment:
We exclude this project because all files in the project are already covered by `restricted_content/sync_up.rb`